src/kernels/installer/moduleInstaller.node.ts

```diff
@@ -113,7 +115,13 @@ export abstract class ModuleInstaller implements IModuleInstaller {
             if (observable) {
                 observable.out.subscribe({
                     next: (output) => {
-                        progress?.report({ message: output.out });
+                        if (progress) {
+                            const message =
+                                output.out.length > 100
+                                    ? `${output.out.substring(0, 50)}...${output.out.substring(output.out.length - 50)}`
+                                    : output.out;
+                            progress?.report({ message });
```

Comment:
Not sure if this actually changes anything, but looks like you added the if check, so the ? wouldn't be needed here anymore.

Reply:
Fixed, no idea why `progress` was marked as nullable, it isn't. Removed the `if` condition (i added that to reduce unnecessary processing)